packages/@sanity/base/src/search/weighted/types.ts

```diff
@@ -1,3 +1,15 @@
+import {ObjectSchemaType} from '@sanity/types'
+
+/**
+ * @internal
+ */
+export interface SearchTerms {
```

Comment:
Making this an object should allow us to extend with new terms in the future (specific fields, dates ect).